.github/actions/install-prometheus/action.yaml

```diff
@@ -0,0 +1,25 @@
+name: InstallPrometheus
+description: 'Installs prometheus'
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+    - name: add prometheus repo
+      shell: bash
+      run: |
+        helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+    - name: create monitoring namespace
+      shell: bash
+      run: |
+        kubectl create ns monitoring || true
+        kubectl label ns monitoring scrape=enabled --overwrite=true
+    - name: install prometheus
+      shell: bash
+      run: |
+        helm upgrade --install prometheus prometheus-community/kube-prometheus-stack \
+        -n monitoring \
+        -f ./.github/actions/install-prometheus/values.yaml \
+        --set "kubelet.serviceMonitor.cAdvisorRelabelings[0].targetLabel=metrics_path" \
```

Comment:
Just for my own understanding: What does metrics path give us here?

Reply:
Honestly, just copied it over from our e2es. Not sure.

Reply:
Hmmm, I honestly don't know. Maybe try dropping it?  I honestly can't remember the context around why I might have added this originally (I assume that I probably did, but I have no idea :) )